spec/hyperlink-spec.coffee

```diff
@@ -31,3 +31,15 @@ describe 'Hyperlink grammar', ->
 
     {tokens} = testGrammar.tokenizeLine 'regexp:http://github.com'
     expect(tokens[1]).toEqual value: 'http://github.com', scopes: ['source.test', 'string.regexp.test']
+
+  describe 'parsing cfml strings', ->
+
+    it 'does not include anything between (and including) \\# signs', ->
```

Comment:
This will actually output as `\#` when running the specs.  Currently there's no way to include `#` in a spec, so you might want to use the word `pound` instead.  Same goes for below.